clients/client-python/gravitino/api/fileset_change.py

```diff
@@ -3,6 +3,9 @@
 This software is licensed under the Apache License version 2.
 """
 from abc import ABC
+from dataclasses import field
+
```

Comment:
Why do we need to add the blank line here?

Reply:
This is Python format.